examples/k8s.yaml

```diff
@@ -125,7 +125,9 @@ provision:
     kubectl taint nodes --all node-role.kubernetes.io/control-plane-
     # Replace the server address with localhost, so that it works also from the host
     sed -e "/server:/ s|https://.*:\([0-9]*\)$|https://127.0.0.1:\1|" -i $KUBECONFIG
-    mkdir -p ${HOME:-/root}/.kube && cp -f $KUBECONFIG ${HOME:-/root}/.kube/config
```

Comment:
This should be retained to keep `sudo kubectl` functional?

Reply:
If needed, it is still possible to use something like:

`sudo KUBECONFIG=/etc/kubernetes/admin.conf kubectl`

https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/

Reply:
Yes, but I wonder if somebody might have been using a script that depends on the old behavior

Reply:
(Same discussion applies to k3s.yaml)

Reply:
I think for k3s it just used the same config file, and just relaxed the file permissions on it. So "should work"